### PR TITLE
feat: Warpify Switch component

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: components/switch/w-switch.vue [hunk 1]
```diff
@@ -1,42 +1,57 @@
+<script setup>
+import { computed, ref } from 'vue'
+import { switchToggle as ccSwitch } from '@warp-ds/component-classes'
+import { createModel, modelProps } from 'create-v-model'
+import { id } from '#util'
+
+const p = defineProps({
```
Comment (CarolineGard) [inline components/switch/w-switch.vue:7]: should we have `props` instead of `p` for readability?

**Review by CarolineGard — APPROVED** [review]
Looking good! 
Just one minor comment

File: components/switch/w-switch.vue (hunk 1 above)
Comment (magnuh) [reply components/switch/w-switch.vue:7]: Using p for props is quite a de facto standard I believe. 😄 I am not sure who I asked about this before... but I think there are a few more components that uses p?

Comment (magnuh) [reply components/switch/w-switch.vue:7]: But it would be a good thing to have it the same everywhere, whatever we choose. 👍

Comment (AnnaRybkina) [reply components/switch/w-switch.vue:7]: I want to remember that I asked you Magnus if `p` was a some sort of Vue non-official standard and I want to remember you said yes 😁

Comment (CarolineGard) [reply components/switch/w-switch.vue:7]: aha okay, there are many components that use `props` as well 😅 then it doesn't matter for now, but we should do the same everywhere later

**Pull request merged**